scripts/globals/job_utils/warrior.lua

```diff
@@ -22,8 +22,13 @@ xi.job_utils.warrior.checkMightyStrikes = function(player, target, ability)
 end
 
 xi.job_utils.warrior.checkTomahawk = function(player, target, ability)
-    --Placeholder code. Needs to check for direction and equiped ammo.
```

Comment:
Confirmed via retail testing that Tomahawk can used while unengaged and while facing any direction.